### ISSUE TITLE
read_raw cannot handle ambiguous file formats

### ISSUE THREAD

**Issue by cbrnr** [cbrnr/mnelab#160]
The current implementation of `read_raw` (soon to be part of MNE) cannot distinguish between file formats that can contain either raw (continuous) or epoched data. For example, .FIF files can store both data types, but currently MNELAB assumes that .FIF files always contain continuous raw data. This should be changed in order to support loading epoched data.

**Comment by abcsds** [comment]
Is this an upstream issue with mne.io.read_raw? I can check.

**Comment by cbrnr** [comment]
This is not an upstream issue, and maybe the title of this issue is not accurate. It is OK that `read_raw` reads only raw (= continuous) files. Epochs are imported with `read_epochs` (only for FIFF files) and `read_epochs_*` for other file types (this is not consistent because `read_epochs` should be named `read_epochs_fif` maybe).

So it's really something that we need to check. If a user opens a FIFF file, we need to determine if it contains raw or epoched data and then call the appropriate function.

**Comment by Mattlab-hub** [comment]
Hi, is this Issue still being worked on?

**Comment by cbrnr** [comment]
Yes, do you want to import epochs from a specific file type? That way, I could prioritize certain file formats.

**Comment by Mattlab-hub** [comment]
Awesome! I just want the ability to preprocess raw data and epoch it, but then read that same epoched data in again later. So the fif format is what's most important. I am using the window version of the app.

**Comment by cbrnr** [comment]
This feature is now available in MNELAB v1.4.5.

**Comment by Mattlab-hub** [comment]
Thank You! It looks like most things work, but I cannot plot the data. I get this error: 
Traceback (most recent call last):
  File "mnelab\mainwindow.py", line 1192, in plot_data
    fig = self.model.current["data"].plot(
        events=events,
    ...<5 lines>...
        show=False,
    )
TypeError: BaseEpochs.plot() got an unexpected keyword argument 'duration'

**Comment by cbrnr** [comment]
Yes, I've already noticed this bug! It's almost fixed, I'll make a new release today or tomorrow!

**Comment by cbrnr** [comment]
OK, this should be fixed in v1.4.6!

**Comment by Mattlab-hub** [comment]
Beautiful! Thank you!
